web_app/config.py

```diff
@@ -0,0 +1,20 @@
+"""Config file."""
+
+import dash  # type: ignore
+import dash_bootstrap_components as dbc  # type: ignore
+import flask
+
+app = dash.Dash(
+    __name__,
+    server=flask.Flask(__name__),
+    external_stylesheets=[
+        dbc.themes.BOOTSTRAP,
+        "https://codepen.io/chriddyp/pen/bWLwgP.css",
+    ],
+)
+
+server = app.server
+app.config.suppress_callback_exceptions = True
+
+DBMS_SERVER_URL = "http://localhost:8080"
+TOKEN_SERVER_URL = "http://localhost:8888"
```

Comment:
Consider getting these from the environment, with this value as default.

Reply:
I'll work in `from_environment()` (https://github.com/WIPACrepo/rest-tools/blob/master/rest_tools/server/config.py#L12)